openapi.yaml

```diff
@@ -12,106 +12,458 @@ info:
+        '500':
+          $ref: '#/components/responses/500'
+    patch:
+      summary: Update details of a specific member.
+      tags:
+        - members
+      description: Update a member by unique identifier.
+      operationId: updateMemberById
+      parameters:
+        - $ref: '#/components/parameters/memberId'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/MemberPatchBody'
+      responses:
+        '200':
+          description: Details of the updated member.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MemberResult'
+        '404':
+          $ref: '#/components/responses/404'
+        '500':
+          $ref: '#/components/responses/500'
+    delete:
+      summary: Remove a specific member of the library.
+      tags:
+        - members
+      description: Remove a member by unique identifier.
+      operationId: deleteMemberById
+      parameters:
+        - $ref: '#/components/parameters/memberId'
+      responses:
+        '204':
+          description: Member successfully deleted.
+        '404':
+          $ref: '#/components/responses/404'
+        '500':
+          $ref: '#/components/responses/500'
+  /borrows:
     get:
-      summary: Info for a specific pet
+      summary: Retrieve a list of all borrowed books from the library.
       tags:
-        - pets
-      description: Get pet by unique ID
-      operationId: getPetById
+        - borrows
+      description: Get all Borrowed Books
+      operationId: getBorrows
       parameters:
-        - name: id
-          in: path
-          description: Unique Pet ID
-          required: true
+        - $ref: '#/components/parameters/pageNumber'
+        - $ref: '#/components/parameters/pageSize'
+        - in: query
+          name: filter[book_id]
+          description: Search by book ID borrowed.
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: filter[member_id]
+          description: Search by member ID who borrowed the book.
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: filter[borrow_date]
+          description: Filter books borrowed on a specific date.
+          required: false
+          schema:
+            type: string
+            format: date
+        - in: query
+          name: filter[due_date]
+          description: Filter books due on a specific date.
+          required: false
+          schema:
+            type: string
+            format: date
+        - in: query
+          name: filter[return_date]
+          description: Filter books returned on a specific date.
+          required: false
+          schema:
+            type: string
+            format: date
+        - in: query
+          name: filter[status]
```

Comment:
How many statuses are available? Can you enumerate them?

Reply:
done